src/library/dlgtagfetcher.cpp

```diff
@@ -250,45 +293,63 @@ void DlgTagFetcher::fetchTagFinished(
 }
 
 void DlgTagFetcher::slotNetworkResult(
-        int httpError, const QString& app, const QString& message, int code) {
+        int httpError,
+        const QString& app,
+        const QString& message,
+        int code) {
     m_networkResult = httpError == 0 ? NetworkResult::UnknownError : NetworkResult::HttpError;
     m_data.m_pending = false;
-    QString strError = tr("HTTP Status: %1");
-    QString strCode = tr("Code: %1");
-    httpStatus->setText(strError.arg(httpError) + "\n" + strCode.arg(code) + "\n" + message);
-    QString unknownError = tr("Mixxx can't connect to %1 for an unknown reason.");
-    cantConnectMessage->setText(unknownError.arg(app));
-    QString cantConnect = tr("Mixxx can't connect to %1.");
-    cantConnectHttp->setText(cantConnect.arg(app));
+
+    if (m_networkResult == NetworkResult::UnknownError) {
+        QString unknownError = tr("Can't connect to %1 for an unknown reason.");
```

Comment:
What makes the UnknownError special?

Reply:
I am not sure actually, It was like this before I started the New GUI. I didn't want to break it down, so I tried make it with the new GUI as it was before. 

After the rate limit bug occurred and the tasks changed a bit, I lost the track of it. I'm sorry.

I would like to find out if it is special case, and have the better user feedback.

Should I work on it?

Reply:
Yes, I suggest do dispose these artificial strings and rely on the strings received in the error message. 
If it turns out that this is ugly. We may adjust them instead.

Reply:
Done +1